src/Commands/InstallCommand.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Rapidez\Statamic\Commands;
+
+use Illuminate\Console\Command;
+
+class InstallCommand extends Command
+{
+    protected $signature = 'rapidez-statamic:install';
+
+    protected $description = 'Install Rapidez Statamic';
+
+    public function handle(): void
+    {
+        $this->call('vendor:publish', [
+            '--tag' => 'runway-config',
+        ]);
+
+        $this->call('vendor:publish', [
+            '--provider' => 'Statamic\Eloquent\ServiceProvider',
+            '--tag' => 'statamic-eloquent-config',
+        ]);
```

Comment:
Maybe ask right away which config the user wants published; https://laravel.com/docs/11.x/artisan#multiple-choice-questions, if the publishing is mandatory, nevermind this comment.

Reply:
Or maybe also include the `Collections, Blueprints and Fieldsets` and `Views` in this? Just add the 4 options and "all" Or we could try making a multiple select like the default `vendor:publish` command.
<img width="629" alt="Scherm­afbeelding 2024-07-30 om 14 49 35" src="https://github.com/user-attachments/assets/0299b792-2d7d-438b-9535-ac12736fe577">

Reply:
Made the publishing semi-mandatory by setting the default to yes, still want to give the choice to the user but make it known that default is to publish.